monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -234,7 +236,11 @@ class ConfigurePageComponent extends AuthComponent {
 
   updateConfigSection = () => {
     let newConfig = this.state.configuration;
-    if (Object.keys(this.state.currentFormData).length > 0) {
+
+    if (this.currentSection === 'attack' && this.state.currentFormData === undefined) {
+      this.state.currentFormData = this.state.attackConfig;
```

Comment:
We don't need `this.state.currentFormData` to be set to `this.state.attackConfig`, because `this.state.attackConfig;` is a separate thing. On attack matrix we don't submit the form data, thus we don't need to track it

Reply:
Then maybe set it to empty ?

Reply:
Yes, I've tried it, but still run into bugs